frontend/src/components/Home/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import styled from "styled-components";
+import Octicon from "octicons-react-ts";
+import { formatDistance } from "date-fns";
+
+import { Page, Container } from "../styles";
+import useApiRequest from "../../hooks/useApi";
+import { Status } from "../../context";
+import UserBadge from "../UserBadge";
+import CoverageTotals from "../CoverageTotals";
+
+export default function Home() {
```

Comment:
This one  has the most internal state, I explained most of it on the call I believe, but essentially we have an api state, repos array, search term (input), as well as two use effect for onload and listening for changes. 

Not that useApiRequest actually is using useReducer but I've been able to abstract that away from Home and Overview so the UI can focus on presentation.